doc/whats_new/v0.23.rst

```diff
@@ -140,6 +142,17 @@ Changelog
    exclusively choose the components that explain the variance greater than
    `n_components`. :pr:`15669` by :user:`Krishna Chaitanya <krishnachaitanya9>`
 
+
+- |Enhancement| added a new approximate solver (randomized SVD, available with
+  `eigen_solver='randomized'`) to :class:`decomposition.KernelPCA`. This
+  drastically accelerates computation when the number of samples gets much
```

Comment:
```suggestion
  accelerates computation when the number of samples is much
```

Reply:
I'll use significantly as suggested in the other thread.